src/Swarm/Game/Step.hs

```diff
@@ -1488,6 +1499,17 @@ execConst c vs s k = do
       let msg = "The operator '$' should only be a syntactic sugar and removed in elaboration:\n"
        in throwError . Fatal $ msg <> badConstMsg
  where
+  installSelf :: (HasRobotStepState sig m, Has (Lift IO) sig m) => Entity -> RID -> RID -> m ()
+  installSelf item focusedID myID = do
```

Comment:
Seems a bit strange to take `focusedID` and `myID` as parameters when all we're going to do is compare them to see if they are equal.  Why not just take a `Bool` instead, called `viewingSelf` or something like that?

Reply:
fixed